zigpy/ota/provider.py

```diff
@@ -535,3 +536,85 @@ async def refresh_firmware_list(self) -> None:
                     self._cache[img.key] = img
 
         self.update_expiration()
+
+
+@attr.s
+class INOVELLIImage:
+    manufacturer_id = attr.ib()
+    image_type = attr.ib()
+    version = attr.ib(default=None)
+    image_size = attr.ib(default=None)
+    url = attr.ib(default=None)
+
+    @classmethod
+    def new(cls, data):
+        mod = data["model"]
+        ver = data["version"]
+        url = data["firmware"]
+
+        res = cls(
+            manufacturer_id=Inovelli.MANUFACTURER_ID, model=mod, version=ver, url=url
+        )
+        return res
+
+    @property
+    def key(self):
+        return ImageKey(self.manufacturer_id, self.image_type)
```

Comment:
Since you don't know the OTA `image_type`, the second element in the `ImageKey` should be the model name.